src/routes/jira/workspaces/jira-workspaces-get.ts

```diff
@@ -1,3 +1,74 @@
-export const JiraWorkspacesGet = () => {
-	// Content included in ARC-2115-search-connected-orgs
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { RepoSyncState } from "models/reposyncstate";
+import { Errors } from "config/errors";
+
+type GitHubWorkspace = {
+	id: string,
+	name: string
+}
+
+const {
+	MISSING_JIRA_HOST,
+	MISSING_SUBSCRIPTION,
+	MISSING_ORG_NAME,
+	NO_MATCHING_WORKSPACES
+} = Errors;
+
+const findMatchingOrgs = async (subscriptions: Subscription[], orgName: string): Promise<GitHubWorkspace[]>  => {
+	const matchingRepos = await RepoSyncState.findByOrgNameAndSubscriptionId(subscriptions, orgName);
+	const matchedOrgs = matchingRepos.map(org => {
+		const { subscriptionId, repoOwner } = org;
+
+		return {
+			id: subscriptionId.toString(),
+			name: repoOwner,
+			// default to false until support is added for createContainer
+			canCreateContainer: false
+		};
+	})
+		.filter((value, index, self) =>
+			index === self.findIndex((org) => (
+				org.id === value.id
+			))
+		);
+
+	return matchedOrgs;
+};
+
+export const JiraWorkspacesGet = async (req: Request, res: Response): Promise<void> => {
+	req.log.info({ method: req.method, requestUrl: req.originalUrl }, "Request started for GET workspaces");
+
+	const { jiraHost } = res.locals;
+
+	if (!jiraHost) {
```

Comment:
I think we can ignore its, since it will/should be verified somewhere up the router?